configuration/concepts/lab_tests.csv

```diff
@@ -1,29 +1,134 @@
-2cbfd68c-b45a-4c25-b9a3-650169d87940,,CIEL:1000,Blood,,,Sample,N/A,,,Rapid HIV; Fasting blood glucose; Random blood glucose; HbA1C; Rapid malaria;,,,,,,,,
-958cd481-53cb-446a-b4c5-9d1dc00ded24,TRUE,,,,,,,,,,,,,,,,,
-09db95e4-185c-4a22-9464-10c77dff27be,,CIEL:159994,Urine,,,Sample,N/A,,,Urinalysis; Urine pregnancy test;,,,,,,,,
-faabf960-2025-42e1-8eed-f7825cfa6e2d,,,Swab,,,Sample,N/A,,,Chlamydia; Gonorrhea; 74ab27c4-c745-45ff-b7e6-e8dbaf933458; b2a75309-4eb5-414d-9f29-8ecb4ac7dd32; Yeast;,,,,,,,,
-ef5d7595-87f6-4814-95e0-41533a517783,,,Others,,,Sample,N/A,,,Miscellaneous; ,,,,,,,,
-,,,Lab Samples,Laboratory,,Misc,N/A,,,Blood; Urine; Swab; Others,,,,,,,,
\ No newline at end of file
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:800
+718b4589-2a11-4355-b8dc-aa668a93e098,,CIEL:664,Negative,Négatif,Negative,Négatif,,,Misc,N/A,,,,,,,,,,,
+7acfafa4-f19b-485e-97a7-c9e002dbe37a,,CIEL:703,Positive,Positif,Positive,Positif,,,Misc,N/A,,,,,,,,,,,
+484bfc3e-2109-47ad-beae-4689d5925f26,,CIEL:1138,Indeterminate,Indeterminate,,,,,Misc,N/A,,,,,,,,,,,
+a24afd60-b6b6-4b7e-82b4-dc599ce62ead,,CIEL:1040,Rapid HIV,Rapid HIV,,,,,LabTest,Coded,,"Positive; Negative;Indeterminate ",,,,,,,,,
+dbff115d-32df-4bd5-8265-832e1f3cbe5e,,CIEL:160912,Fasting blood glucose,Fasting blood glucose,,,,,LabTest,Numeric,,,,Yes,mg/dL,70,126,,,,
+3a7aefbf-3334-4add-88a0-42ee1c6fd8dd,,CIEL:887,Random blood glucose,Random blood glucose,,,,,LabTest,Numeric,,,,Yes,mg/dL,70,180,,,,
+df5686c1-ad0c-4deb-9919-f77446fd5bb9,,CIEL:159644,HbA1C,HbA1C,,,,,LabTest,Numeric,,,,Yes,%,4,6.5,,,,
+3f330744-e90c-4a32-9048-07dd255782eb,,CIEL:1643,Rapid malaria,Rapid malaria,,,,,LabTest,Coded,,"Positive; Negative; Indeterminate ",,,,,,,,,
+571e8f4f-4d98-40eb-bd55-c88b8a8ab76d,,CIEL:45,Urine pregnancy test,Urine pregnancy test,,,,,LabTest,Coded,,"Positive; Negative; Indeterminate ",,,,,,,,,
+b2a75309-4eb5-414d-9f29-8ecb4ac7dd32,,CIEL:161456,Bacterial vaginosis,Bacterial vaginosis,,,,,LabTest,Coded,,Positive; Negative; Indeterminate,,,,,,,,,
+74ab27c4-c745-45ff-b7e6-e8dbaf933458,,CIEL:163648,Trichomonas,Trichomonas,,,,,LabTest,Coded,,Positive; Negative; Indeterminate,,,,,,,,,
+14e99cf4-d294-4905-8837-40e704d8a6ad,,,Vaginal Candidiasis,Vaginal Candidiasis,,,,,LabTest,Coded,,Positive; Negative; Indeterminate,,,,,,,,,
+ab6588f6-87ee-45ca-a6c5-dc1337f83469,,CIEL:1036,Gonorrhea,Gonorrhea,,,,,LabTest,Coded,,Positive; Negative; Indeterminate,,,,,,,,,
+637b4f48-8717-45e2-a5f8-96869e3bb768,,CIEL:163618,Chlamydia,Chlamydia,,,,,LabTest,Coded,,Positive; Negative; Indeterminate,,,,,,,,,
+4369ee0e-5b3d-4af4-b28b-5cc5cfc34d91,,,Miscellaneous,Miscellaneous,,,,,LabTest,Text,,,,,,,,,,,
+ef5d7595-87f6-4814-95e0-41533a517783,,,Other tests,Other tests,,,,,Sample,N/A,,,Miscellaneous;,,,,,,,,

```

Comment:
Let's remove those tests that were provided in Haiti. We'll replace them when merging with HSC lab tests.

Reply:
Done